databases/wrath-auth-db/src/lib.rs

```diff
@@ -28,105 +30,111 @@ impl AuthDatabase {
+        Ok(sqlx::query_as::<_, DBRealm>("SELECT * FROM realms")
+            .fetch_all(&self.connection_pool)
+            .await?)
     }
 
     pub async fn set_realm_online_status(&self, realm_id: u32, online: bool) -> Result<()> {
-        sqlx::query!("UPDATE realms SET online = ? WHERE id = ?", online as u8, realm_id)
+        sqlx::query("UPDATE realms SET online = ? WHERE id = ?")
+            .bind(online as u8)
+            .bind(realm_id)
             .execute(&self.connection_pool)
             .await?;
         Ok(())
     }
 
     pub async fn set_realm_population(&self, realm_id: u32, population: f32) -> Result<()> {
-        sqlx::query!("UPDATE realms SET population = ? WHERE id = ?", population, realm_id)
+        sqlx::query("UPDATE realms SET population = ? WHERE id = ?")
+            .bind(population)
+            .bind(realm_id)
             .execute(&self.connection_pool)
             .await?;
         Ok(())
     }
 
     pub async fn get_account_by_username(&self, username: &str) -> Result<DBAccount> {
-        let acc = sqlx::query_as!(DBAccount, "SELECT * FROM accounts WHERE username = ?", username)
+        let acc = sqlx::query_as::<_, DBAccount>("SELECT * FROM accounts WHERE username = ?")
+            .bind(username)
             .fetch_one(&self.connection_pool)
             .await?;
         Ok(acc)
     }
 
     pub async fn set_account_v_s(&self, account_id: u32, v: &str, s: &str) -> Result<()> {
-        sqlx::query!("UPDATE accounts SET v = ?, s = ? WHERE id = ?", v, s, account_id)
+        sqlx::query("UPDATE accounts SET v = ?, s = ? WHERE id = ?")
+            .bind(v)
+            .bind(s)
+            .bind(account_id)
             .execute(&self.connection_pool)
             .await?;
 
         Ok(())
     }
 
     pub async fn set_account_sessionkey(&self, username: &str, session_key: &str) -> Result<()> {
-        sqlx::query!("UPDATE accounts SET sessionkey = ? WHERE username = ?;", session_key, username)
+        sqlx::query("UPDATE accounts SET sessionkey = ? WHERE username = ?;")
+            .bind(session_key)
+            .bind(username)
             .execute(&self.connection_pool)
             .await?;
         Ok(())
     }
 
     pub async fn get_account_data(&self, account_id: u32) -> Result<Vec<DBAccountData>> {
-        let acc_data = sqlx::query_as!(DBAccountData, "SELECT * FROM account_data WHERE account_id = ?", account_id)
+        let acc_data = sqlx::query_as::<_, DBAccountData>("SELECT * FROM account_data WHERE account_id = ?")
+            .bind(account_id)
             .fetch_all(&self.connection_pool)
             .await?;
         Ok(acc_data)
     }
 
     pub async fn get_account_data_of_type(&self, account_id: u32, data_type: u8) -> Result<DBAccountData> {
-        Ok(sqlx::query_as!(
-            DBAccountData,
-            "SELECT * FROM account_data WHERE account_id = ? AND data_type = ?",
-            account_id,
-            data_type
+        Ok(
```

Comment:
please revert to macro variant query_as!

Reply:
reverted